cli/install/install.go

```diff
@@ -196,7 +197,8 @@ func detectOsName() (string, error) {
 
 // getVersionsFromRepo returns all available versions from github repository.
 func getVersionsFromRepo(local bool, distfiles string, program string,
-	repolink string) ([]version.Version, error) {
+	repolink string,
+) ([]version.Version, error) {
```

Comment:
We don't need this unnecessary changes. Especially in the commit.

Please, revert all unnecessary/code-style changes unrelated to the issue. This only makes it more difficult to view the change history without any value. Changes in code style must be accepted for the entire project and covered by a linter.